check.go

```diff
@@ -99,5 +99,5 @@ func (d *DBFT) checkChangeView(view byte) {
 		}
 	}
 
-	d.InitializeConsensus(view)
+	d.InitializeConsensus(view, 0)
```

Comment:
Why don't we reset it to the last accepted block timestamp?

Reply:
It stays the same in fact, but yes, we can have `d.InitializeConsensus(view, d.lastBlockTimestamp)` here as well.

Reply:
Oh, I missed that `InitializeConsensus` updates timestamp only on 0 view.

Reply:
I agree, `d.lastBlockTimestamp` is more straightforward and allows us to avoid `if` inside `InitializeConsensus`.

Reply:
Adjusted a bit (and moved this assignment to `Context`, that's where the field is anyway).